api/migrations/20210812174148-create-guild.js

```diff
@@ -12,27 +12,31 @@ module.exports = {
         defaultValue: ".",
         type: Sequelize.STRING,
       },
+      defaultMemberRoles: {
+        defaultValue: [],
+        type: Sequelize.ARRAY(Sequelize.STRING),
+      },
       opRoles: {
         defaultValue: [],
         type: Sequelize.ARRAY(Sequelize.STRING),
       },
       levels: {
         defaultValue: "[]",
-        type: Sequelize.STRING,
+        type: Sequelize.STRING(1024),
```

Comment:
Does this limit string to 1024 characters?

Reply:
I have changed the type to Sequelize.TEXT which doesn't limit the string in length.
https://sequelize.org/master/class/lib/data-types.js~TEXT.html